src/adr/0051-pulsar-mailqueue.md

```diff
@@ -0,0 +1,168 @@
+# 51. Pulsar MailQueue
```

Comment:
It should be 52
We have duplicated 50 (0050-oidc.md , 0050-jmap-web-push.md)
I fixed that https://github.com/apache/james-project/pull/833/commits/4c0bd229b80796b32240cb2607aab16cdf075c31